packages/valory/services/trader/service.yaml

```diff
@@ -104,6 +104,11 @@ type: skill
         redeeming_batch_size: ${REDEEMING_BATCH_SIZE:int:1}
         slippage: ${SLIPPAGE:float:0.01}
         redeem_margin_days: ${REDEEM_MARGIN_DAYS:int:15}
+        epsilon: ${EPSILON:float:0.1}
```

Comment:
What is this?

Reply:
For the epsilon greedy policy.

Reply:
please use a more descriptive name if possible and don't require significant refactoring.

Reply:
Renamed on fb597753a6657296a20048dd42e831b4e6d9b781.